bright-ifit-purchase-validation.yaml

```diff
@@ -33,6 +33,24 @@ components:
           example: roku.tv.ifit.beta
       discriminator:
         propertyName: type
+    AndroidTVValidateSubscriptionRequest:
+      allOf:
+        - $ref: '#/components/schemas/ValidateSubscriptionRequest'
+        - type: object
+          required:
+            - applicationId
+            - productId
+            - purchaseToken
+          properties:
+            type:
+              type: string
+              example: android
+            productId: # might be subscriptionId
+              type: string
+              example: ifryme
```

Comment:
TODO I'm clarifying this with Bright AI [in Slack](https://ifit.slack.com/archives/C01N3U2FD2M/p1620770686146700?thread_ts=1619645130.093500&cid=C01N3U2FD2M)

Reply:
btw this was resolved when [I asked during TV scrum today](https://ifit.slack.com/archives/C01N3U2FD2M/p1620928428156700?thread_ts=1619645130.093500&cid=C01N3U2FD2M) - for now, iFit will expect `subscriptionId` from Bright.AI